forwarder.go

```diff
@@ -10,24 +10,44 @@ import (
 	"net/http"
 	"os"
 	"strings"
+	"sync"
 )
 
+const workers = 8
+
 func main() {
 	log.Println("Splunk forwarder: Started")
 	defer log.Println("Splunk forwarder: Stopped")
 
+	forSplunk := make(chan string)
+
+	var wg sync.WaitGroup
+
+	for i := 0; i < workers; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for msg := range forSplunk {
+				postToSplunk(msg)					
+			}
+		}()
+	}
+
 	br := bufio.NewReader(os.Stdin)
 	for {
 		str, err := br.ReadString('\n')
```

Comment:
@khatton-ft @martingartonft 
The following commit was made to remove all references to WaitGroup.
https://github.com/Financial-Times/coco-splunk-http-forwarder/pull/6/commits/f0f6a6674d1dba36b024120afa23cbaf79dc3f10

Code has been validated with the following test and results are good.

cat ~/Documents/ft/containers/coco-splunk-http-forwarder/journald.sample | go run ./forwarder.go -url=https://ingest.splunk.glb.ft.com/coco-up/fleet
2016/09/28 16:33:03 Splunk forwarder: Started
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Event sent to splunk
2016/09/28 16:33:03 Splunk forwarder: Stopped